### ISSUE TITLE
TypeError: Path must be a string. Received undefined                                                                        

### ISSUE THREAD

**Issue by k1ng440** [fullcube/loopback-component-model-extender#1]
just installed this component and i am getting 

```
TypeError: Path must be a string. Received undefined                                                                                                                                                                                             
    at assertPath (path.js:7:11)                                                                                                                                                                                                                 
    at Object.join (path.js:466:7)                                                                                                                                                                                                               
    at loadComponents (D:\projects\eleven\ElevenBackend2\node_modules\loopback-component-model-extender\index.js:51:27)                                                                                                                          
    at Object.keys.forEach.key (D:\projects\eleven\ElevenBackend2\node_modules\loopback-component-model-extender\index.js:70:5)                                                                                                                  
    at Array.forEach (native)                                                                                                                                                                                                                    
    at module.exports (D:\projects\eleven\ElevenBackend2\node_modules\loopback-component-model-extender\index.js:69:27)                                                                                                                          
    at D:\projects\eleven\ElevenBackend2\node_modules\loopback-boot\lib\executor.js:437:5                                                                                                                                                        
    at Array.forEach (native)                                                                                                                                                                                                                    
    at setupComponents (D:\projects\eleven\ElevenBackend2\node_modules\loopback-boot\lib\executor.js:430:27)                                                                                                                                     
    at execute (D:\projects\eleven\ElevenBackend2\node_modules\loopback-boot\lib\executor.js:41:3)                                                                                                                                               
    at bootLoopBackApp (D:\projects\eleven\ElevenBackend2\node_modules\loopback-boot\index.js:154:3)                                                                                                                                             
    at Object.<anonymous> (D:\projects\eleven\ElevenBackend2\server\server.js:18:1)                                                                                                                                                              
    at Module._compile (module.js:556:32)                                                                                                                                                                                                        
    at Object.Module._extensions..js (module.js:565:10)                                                                                                                                                                                          
    at Module.load (module.js:473:32)                                                                                                                                                                                                            
    at tryModuleLoad (module.js:432:12)                                                                                                                                                                                                          

```

EDIT:
that error occurs when basePath is not defined in component-config.json

**Comment by beeman** [comment]
Hi @k1ng440, thanks for creating this issue. I'll get a fix in. 

In the meanwhile you can work around it by defining the component like this:

``` json
"loopback-component-model-extender": {
  "basePath": "."
}
```

**Comment by mrfelton** [comment]
@k1ng440 this is fixed in 0.1.3. Thanks for reporting.
